vite.config.ts

```diff
@@ -5,27 +5,27 @@ import { loadSemcoreSources } from './website/docs/.vitepress/load-semcore-sourc
 import pluginReact from '@vitejs/plugin-react';
 
 export default defineConfig({
-    plugins: [
-        pluginReact(),
-        createUnplugin<{}>(() => ({
-            name: 'semcore-resolve',
-            async resolveId(id) {
-                if (
-                    !id.includes('@semcore') &&
-                    !id.includes('/semcore/') &&
-                    !id.startsWith('intergalactic/')
-                )
-                    return null;
-                if (id.endsWith('.md') || id.endsWith('.mdx') || id.includes('stories')) return null;
-                return await resolveSemcoreSources(id);
-            },
-            loadInclude: (id) => {
-                return id.includes('/semcore/');
-            },
-            async load(id) {
-                return await loadSemcoreSources(id);
-            },
-            enforce: 'pre',
-        })).vite({}),
-    ],
+  plugins: [
+    pluginReact(),
+    createUnplugin<{}>(() => ({
+      name: 'semcore-resolve',
+      async resolveId(id) {
+        if (
+          !id.includes('@semcore') &&
+          !id.includes('/semcore/') &&
+          !id.startsWith('intergalactic/')
+        )
+          return null;
```

Comment:
Is this one - line `if` works as expected with line break after condition?

Reply:
Maybe I didn't quite understand you, but I think the answer is yes.